src/background/actions/shortcuts-page-action-creator.ts

```diff
@@ -18,8 +19,8 @@ export class ShortcutsPageActionCreator {
         this.interpreter.registerTypeToPayloadCallback(Messages.Shortcuts.ConfigureShortcuts, this.onConfigureShortcuts);
     }
 
-    private onConfigureShortcuts = (payload: BaseActionPayload): void => {
-        this.shortcutsPageController.openShortcutsTab();
+    private onConfigureShortcuts = async (payload: BaseActionPayload): Promise<void> => {
+        await this.shortcutsPageController.openShortcutsTab();
```

Comment:
This has the side effect of preventing the telemetry from being sent in the event that opening the shortcuts tab encounters an error. Is that desirable?

Reply:
Good catch!

Reply:
Updated.